insights/parsers/freeradius.py

```diff
@@ -0,0 +1,53 @@
+"""
+FreeRadius - Freeradius configuration files
+===========================================
+
+FreeRadius is a parser for the freeradius configuration files usually in
+``/etc/raddb``.
+Files ending with ``.rpmnew`` are excluded because they are backups made when
+ugprading but customer modified the configuration files, so are not used by the
+service.
+
+Because this parser reads multiple files, the files are stored as a list within
+the parser and need to be iterated through in order to find specific
+configuration files.
+
+Examples:
+
+    >>> for cfgfile in shared[FreeRadius]: # Parser contains list of all configuration files
+    ...     print "Path:", cfgfile.file_path
+    ...
+    Path: etc/raddb/radiusd.conf
+    Path: etc/raddb/proxy.conf
+    Path: etc/raddb/policy.d/filter
+
```

Comment:
Really _REALLY_ needs better examples, mainly detailing what is in each `cfgfile` and how to get configuration information.

Reply:
Then you can test it using the `doctest` framework - see the examples in http://insights-core.readthedocs.io/en/1.x/docs_guidelines.html.  I would recommend also using the doctest-as-test system - I've got https://github.com/RedHatInsights/insights-core/pull/845 as a merge to improve the documentation update but examples can be found in #817.

That way you the examples can also be your tests!